test/jdk/javax/swing/JMenu/4213634/bug4213634.java

```diff
@@ -39,85 +39,79 @@
  * @test
  * @key headful
  * @bug 4213634 8017187
- * @author Scott Violet
  * @library ../../regtesthelpers
  * @build Util
+ * @summary Verifies if Alt+mnemonic char works when
+ *          menu & menuitem have same mnemonic char
  * @run main bug4213634
  */
 
-
 public class bug4213634 {
 
-    private JMenu menu;
-
-    private JFrame frame;
-
-    public static void main(String[] args) throws Throwable {
-        new bug4213634();
-    }
+    private static JMenu menu;
+    private static JFrame frame;
+    private static Robot robot;
 
-    bug4213634() throws AWTException, InterruptedException, InvocationTargetException {
-        SwingUtilities.invokeAndWait(new Runnable() {
-            @Override
-            public void run() {
-                createAndShowGUI();
-            }
-        });
+    public static void main(String[] args) throws Exception {
+        try {
+            robot = new Robot();
+            SwingUtilities.invokeAndWait(() -> createAndShowGUI());
```

Comment:
```suggestion
            SwingUtilities.invokeAndWait(bug4213634::createAndShowGUI);
);
```
A method reference can be used.